tests/qunit-adapter.js

```diff
@@ -0,0 +1,15 @@
+import { QUnitAdapter } from 'ember-qunit';
+import Ember from 'ember';
+
+const useEmberOnError = true;
```

Comment:
Here we allow tests to user Ember's `onerror` instead of having errors get swallowed by qunit's exception.